errors/no-before-interactive-script-outside-document.md

```diff
@@ -1,13 +1,15 @@
-# No Before Interactive Script Outside Document
+# No Before Interactive Script Outside Document Or Root Layout
 
-> Prevent usage of `next/script`'s `beforeInteractive` strategy outside of `pages/_document.js`.
+> Prevent usage of `next/script`'s `beforeInteractive` strategy outside of `pages/_document.js` or root `layout.js`.
 
 #### Why This Error Occurred
 
-You cannot use the `next/script` component with the `beforeInteractive` strategy outside `pages/_document.js`. That's because `beforeInteractive` strategy only works inside **`pages/_document.js`** and is designed to load scripts that are needed by the entire site (i.e. the script will load when any page in the application has been loaded server-side).
+You cannot use the `next/script` component with the `beforeInteractive` strategy outside `pages/_document.js`. That's because `beforeInteractive` strategy only works inside **`pages/_document.js`** and **root `layout.js`** and is designed to load scripts that are needed by the entire site (i.e. the script will load when any page in the application has been loaded server-side).
```

Comment:
There's no limitation in `app` on where you can use `beforeInteractive` on `next/script`, so this rule can be disabled for files in `app` 👍

Reply:
Cool. It's still tricky. Since you can import code from outside of `app/` in app router.

So we might want to stop enforcing this outside of `pages/` when we stabilize the app router?